src/apps/eth/eth_params.c

```diff
@@ -13904,22 +13895,13 @@ const app_eth_erc20_params_t* app_eth_erc20_params_get(
     uint64_t chain_id,
     const uint8_t* contract_address)
 {
-    const app_eth_erc20_params_t* erc20_params;
     size_t len;
-    switch (chain_id) {
-    case 1:
-        erc20_params = _ethereum_erc20_params;
-        len = sizeof(_ethereum_erc20_params) / sizeof(app_eth_erc20_params_t);
-        break;
-    case 3:
-        erc20_params = _ropsten_erc20_params;
-        len = sizeof(_ropsten_erc20_params) / sizeof(app_eth_erc20_params_t);
-        break;
-    default:
+    if (chain_id != 1) {
         return NULL;
     }
+    len = sizeof(_ethereum_erc20_params) / sizeof(app_eth_erc20_params_t);
```

Comment:
`size_t len = ...`, no need to declare the var separately above.

Could also just inline it in the for-statement.

Reply:
Good point, thx! Done